tests/test_download.py

```diff
@@ -109,3 +112,58 @@ def test_initialize_on_asset_with_custom_credentials(
     mint_tokens_and_wait(dt_contract, consumer_wallet, publisher_wallet)
 
     send_order(client, ddo, dt_contract, sa, consumer_wallet, expect_failure=True)
+
+
+def test_download_multiple_files(client, publisher_wallet, consumer_wallet):
+    ddo = get_dataset_ddo_with_multiple_files(client, publisher_wallet)
+    dt_token = DataToken(ddo.data_token_address)
+
+    mint_tokens_and_wait(dt_token, consumer_wallet, publisher_wallet)
+
+    sa = ddo.get_service(ServiceTypes.ASSET_ACCESS)
+    tx_id = send_order(client, ddo, dt_token, sa, consumer_wallet)
+
+    # Consume using url index and auth token
+    # (let the provider do the decryption)
+    payload = {
+        "documentId": ddo.did,
```

Comment:
Since these payloads only differ by fileIndex, can we please not copy/paste and assign? Instead, let's make one request -> change the body -> do the second etc.

Reply:
@mariacarmina I mean you can change just the keys that need to be changed. I think fileIndex and signature. The rest of the payload stays the same, so no need to redeclare it. Just update the 2 changed keys.